lib/src/algorithms/rsa.dart

```diff
@@ -9,8 +9,8 @@ abstract class AbstractRSA {
   final AsymmetricBlockCipher _cipher;
 
   AbstractRSA({
-    this.publicKey,
-    this.privateKey,
+    required this.publicKey,
```

Comment:
Same happens here. What was the original intent?

Is `publicKey` only required for encrypting and `privateKey` only for decrypting? Should it be possible to pass only one of those and only call the respective method or do both need to be present always?

Reply:
There are some feature requests for encrypting and decrypting with both keys, so let make them optional for now.